test/e2e/storage/subpath.go

```diff
@@ -78,6 +79,120 @@ var _ = utils.SIGDescribe("Subpath", func() {
 
 	f := framework.NewDefaultFramework("subpath")
 
+	Context("Atomic writer volumes", func() {
+		var err error
+
+		BeforeEach(func() {
```

Comment:
nit - Do we need an AfterEach to cleanup the secret and configmap? How about the pods? (where do they get cleaned up?)

Reply:
the entire namespace gets deleted at the end of the test

Reply:
thanks @liggitt !